README.md

```diff
@@ -84,6 +84,8 @@ else fails, try removing the virtual environment and reinstalling:
 
 ## Development
 
+- `cd` into `webpack`
+- Run `npm install` to install required node packages
```

Comment:
I don't recall needing to run these commands for starting the dev server. Is it really needed?

Reply:
Hmm..Well, this sounds weird as it posed a huge issue for me yesterday, while trying to set up, before @dhruvkb pointed out running `npm install` explicitly which instantly got everything working.

Reply:
When the Node.js version was v12, Lektor build automatically installed Node packages for me. I think this step can be omitted, provided we state the Node.js version requirement quite clearly beforehand.

Reply:
This makes sense now, it failed at first because i had not met requirements. So explicitly stating Node v12 as a prerequisite would automatically null that step.